src/Composer/Installers/RadPHPInstaller.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Composer\Installers;
+
+class RadPHPInstaller extends BaseInstaller
+{
+    protected $locations = array(
+        'bundle' => 'bundles/{$name}/'
```

Comment:
@niksamokhvalov To satisfy your curiosity, bundles are used for automatic-routing which contain actions (something like controllers). So similar routing for similar action is not meaningful! For example you cannot have two routing for Pages bundle.

All other libraries go to default directory.